ckanext/keycloak/views.py

```diff
@@ -43,16 +31,16 @@ def _log_user_into_ckan(resp):
     """
     if tk.check_ckan_version(min_version="2.10"):
         from ckan.common import login_user
-        login_user(g.userobj)
+        login_user(g.user)
         return
 
     if tk.check_ckan_version(min_version="2.9.6"):
-        user_id = "{},1".format(g.userobj.get('id'))
+        user_id = "{},1".format(g.user.id)
     else:
-        user_id = g.userobj['name']
+        user_id = g.user.name
```

Comment:
is this right? AFAIK `g.user` should still contain only the username and `g.userobj` should be the object

Reply:
Hey @tino097 thanks for your feedback. If you check the line 84 and 85 from the previous change, the variables `g.user` and `g.userobj` had the same value, the output of function `helpers.process_user(user_dict)`, but `g.user` is never used.

I removed `g.userobj` and kept `g.user`, as I thought it would be redundant naming the field as user_obj, because it is always an object. Do yo think it should be named `g.userobj` instead?

Reply:
@brunopacheco1 as you can check in the [core](https://github.com/ckan/ckan/blob/70f1ee5d7c94cf121cd1c4562dfd325f5c52f339/ckan/views/user.py#L49) we are sending the object to flask's `login_user` so yes, you should pass the `g.userobj` 


But this is for 2.10+ CKAN versions

Reply:
@tino097 I have renamed the variables to keep the same convention as before, but I fixed the `g.user` to be the `g.user_obj.name`, as you suggested to be.